spec/ParseUser.spec.js

```diff
@@ -1265,6 +1265,35 @@ describe('Parse.User testing', () => {
     done();
   });
 
+  fit('log in with Facebook and save signed up User with verifyUserEmails=true and preventLoginWithUnverifiedEmail=true', async () => {
+    const emailAdapter = {
+      sendPasswordResetEmail: () => Promise.resolve(),
+      sendMail: () => Promise.resolve(),
+    };
+    await reconfigureServer({
+      appName: 'unused',
+      verifyUserEmails: true,
+      preventLoginWithUnverifiedEmail: true,
+      emailAdapter: {
+        module: emailAdapter,
+      },
+      publicServerURL: 'http://localhost:8378/1',
+    });
+
+    const provider = getMockFacebookProvider();
+    Parse.User._registerAuthenticationProvider(provider);
+    const user = await Parse.User._logInWith('facebook');
+    expect(user instanceof Parse.User).toBeTrue();
+    expect(Parse.User.current()).toEqual(user);
+    expect(user.extended()).toBeTrue();
+    expect(provider.authData.id).toBe(provider.synchronizedUserId);
+    expect(provider.authData.access_token).toBe(provider.synchronizedAuthToken);
+    expect(provider.authData.expiration_date).toBe(provider.synchronizedExpiration);
+    expect(user._isLinked('facebook')).toBeTrue();
+
+    await expectAsync(user.save());
```

Comment:
There is something missing here. Please take a look at the other usages of expectAsync, like .toBeResolved().

Reply:
Looked for toBeResolved() in other tests.I changed that line to `await expectAsync(user.save()).toBeResolved();`

If this is okay then will commit the file.
Below is test log

```
Parse.User testing
    ✗ log in with Facebook and save signed up User with verifyUserEmails=true and preventLoginWithUnverifiedEmail=true
      - Expected [object Promise] to be resolved.



Failures:
1) Parse.User testing log in with Facebook and save signed up User with verifyUserEmails=true and preventLoginWithUnverifiedEmail=true
  Message:
    Expected [object Promise] to be resolved.
  Stack:
    Error
        at <Jasmine>
        at UserContext.<anonymous> (/Users/ashishn/Developer/GitHub Projects/Parse-Server-Fix-Facebook-Signup-Error206/spec/ParseUser.spec.js:1294:36)
        at processTicksAndRejections (node:internal/process/task_queues:96:5)

Ran 1 of 161 specs
1 spec, 1 failure
Finished in 0.581 seconds
Randomized with seed 15146 (jasmine --random=true --seed=15146)
**************************************************
*                    Failures                    *
**************************************************

1) Parse.User testing log in with Facebook and save signed up User with verifyUserEmails=true and preventLoginWithUnverifiedEmail=true
  - Expected [object Promise] to be resolved.

Executed 1 of 161 specs (1 FAILED) (160 SKIPPED) in 0.581 sec.
Randomized with seed 15146.
```